app/views/usages/add.html.erb

```diff
@@ -0,0 +1,17 @@
+<% provide(:title, 'Usage') %>
+<h1>Select Subscriptions</h1>
+<table class='table'>
+  <tr>
+    <th>Subscription Date</th>
+    <th>User</th>
+    <th>Plan</th>
+    <th>Active</th>
+    <th>Actions</th>
+  </tr>
+  <tbody class='tbody'>
+  <% @subscriptions.each { |subscription| %>
+    <%=  render 'usages/subscription', subscription: subscription  %>
+  <% } %>
```

Comment:
pass the whole `@subscriptions` list to the partial and then do the iteration there.